generators/import-jdl/index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
-const util = require('util');
-const shelljs = require('shelljs');
-const generator = require('yeoman-generator');
-const chalk = require('chalk');
-const jhiCore = require('jhipster-core');
-const scriptBase = require('../generator-base');
+
+const util = require('util'),
```

Comment:
This was changed on purpose as its the most commonly used code style to have const and let defined as separate lines. Its also the default in most of the eslint style templates. So i guess we should also stick to it. I was gonna updade eslint rules for this as well